### PR TITLE
zive.cz, mobilmania.cz

### PR DESCRIPTION
**Pull request by Hakatitla**

### CODE REVIEW

File: filters.txt [hunk 1]
```diff
@@ -288,7 +288,7 @@ _clickthru.swf?
 ||romea.cz/images/servis/banner
 ||ronnie.cz/cache/bannery/
 ||rubiconproject.com$third-party
-||imedia.cz/$script
```
Comment (addisr) [inline filters.txt]: Is there any reason for this revert?

File: filters.txt [hunk 2]
```diff
@@ -1047,14 +1047,18 @@ minecrafak.cz###sidebar
...
-mobilmania.cz##div:-abp-has(> #Leaderboard-1)
 mobilmania.cz##iframe[src*="alza"]
 mobilmania.cz##noscript + script + div
 mobilmania.cz##table#prolinkhdr-v3 + div:not(.cs-cnt-wrapper):not(#mainwrap)
+mobilmania.cz###content-lead
+mobilmania.cz,zive.cz###Leader-1
+mobilmania.cz,zive.cz###bgTop-iFrame
+mobilmania.cz,zive.cz##.redirectArea
+mobilmania.cz,zive.cz##div[id$="_overide"]
+mobilmania.cz,zive.cz##div[class$="_chaoz"]
```
Comment (addisr) [inline filters.txt]: Also where do you see all these elements? I don't get any of these.

File: filters.txt [hunk 3]
```diff
@@ -287,8 +287,8 @@ _clickthru.swf?
 ||romea.cz/296x186.swf
 ||romea.cz/images/servis/banner
 ||ronnie.cz/cache/bannery/
-||rubiconproject.com$third-party,domain=aktualne.cz
-||imedia.cz/$script
+||rubiconproject.com$third-party
+||s.imedia.cz/js/szn-script.js
```
Comment (addisr) [inline filters.txt]: Is there any reason for this revert?

File: filters.txt (hunk 2 above)
Comment (petersaly) [reply filters.txt]: image

File: filters.txt (hunk 3 above)
Comment (petersaly) [reply filters.txt]: I made wrong merge. Already corrected it.

Comment (addisr) [reply filters.txt]: This is a last commit, previous rule (`||imedia.cz/$script`) is the correct one.

Comment (petersaly) [reply filters.txt]: Corrected. Thanks for review

**Comment by Aslanex** [comment]
Pardon, je to tam.

**Pull request merged**